src/app/app.component.ts

```diff
@@ -79,7 +79,24 @@ interface ILfSelectedFolder {
 })
 export class AppComponent implements AfterViewInit {
   config = config;
+  signinoption = "redirect";
 
+  get buttontext() {
+    return this.loginComponent?.nativeElement.state === LoginState.LoggedIn
+      ? 'Sign Out with popup'
+      : 'Sign In with popup';
+  }
+
+  openLogin() {
```

Comment:
- move function below ngAfterViewInit (kind of like constructor) just for readability
- remove comment